federated-identity/src/main/java/com/iluwatar/federatedIdentity/Consumer.java

```diff
@@ -0,0 +1,175 @@
+package com.iluwatar.federatedIdentity;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import org.apache.http.client.fluent.Request;
+
+/**
+ * act as consumer.
```

Comment:
I'm not that familiar with Javalin framework but it seems there is no main entry point in this example? And no instructions for the user how the example implements pattern. I would propose to use the same approach as we have in the other examples that there is an app that is run by the user and it fulfills the pattern.